jekyll/_cci2/getting-started.md

```diff
@@ -288,6 +288,12 @@ cd <directory_name>  # change directory to the <directory_name> directory
 cat <file_name>      # show me the contents of the file <file_name>
 ```
 
+## Collaborating with Teammates
+
+It is easy to bring others into into a workflow such as described above.
```

Comment:
```suggestion
It is easy to bring others into a workflow such as described above.
```

Reply:
Just wondering actually - is "workflow" being used here in the general, rather than CCI sense? if so maybe we can phrase differently? Something like - 'It's simple for others to view and follow your projects'?

Reply:
Yes - _workflow_ is being used in general (poor choice, haha!) I think I better use your suggestion!